src/components/Navbar.js

```diff
@@ -1,52 +1,32 @@
-import { Link } from 'react-router-dom';
-import whiteLogo from '../imgs/white-logo.svg';
+import React from "react";
+import "../css/navbar.css";
+import { Link } from "react-router-dom";
+import whiteLogo from "../imgs/LOGO.svg";
+import { LoginCurve, ShoppingCart } from "iconsax-react";
 
 export default function Navbar() {
-	return (
-		<nav className='navbar w3-red navbar-expand-sm navbar-dark bg-dark fixed-top'>
-			<div className='container-fluid'>
-				<a className='navbar-brand' href='/'>
-					<img className='logo' src={whiteLogo} />
-				</a>
-				<button
-					className='navbar-toggler'
-					type='button'
-					data-bs-toggle='collapse'
-					data-bs-target='#collapsibleNavbar'
-				>
-					<span className='navbar-toggler-icon'></span>
-				</button>
+  return (
+    <nav className="w3-bar w3-black">
+      <Link href="#" className="w3-bar-item w3-button">
```

Comment:
Todo: Change the href to the route page like `href="/"` and like this

Reply:
You mean `to="/"`